### ISSUE TITLE
Support “value” scope targeting `const` keyword

### ISSUE THREAD

**Issue by pokey** [cursorless-dev/cursorless#1256]
In the following:

```ts
const aaa: string = "hello"
```

Saying `”take value cap”` should target `"hello"`

[Slack Message](https://talonvoice.slack.com/archives/C026KPTJE6T/p1675717859385399)

**Comment by auscompgeek** [comment]
I presume it should bail if multiple consts are declared in the same statement?

**Comment by nathanheffley** [comment]
@auscompgeek I'm not sure how the tree-sitter looks for that expression, but would it be possible for `take value air` to select the `1` and `take value bat` to select the `2`? That's what I'd expect. If you tried to use a token from `const` bailing seems reasonable.

**Comment by auscompgeek** [comment]
@nathanheffley yep, that's what I would expect to happen with that too. I believe that's the current behaviour there.

**Comment by pokey** [comment]
I guess it could take both of them?  That's how it works with "unit" if there are multiple units in css, eg `border-width: 10px 1px;`

**Comment by AndreasArvidsson** [comment]
This is now working
